tests/composer/autocomplete_dropdown_component.test.ts

```diff
@@ -257,6 +269,58 @@ describe("Functions autocomplete", () => {
         fixture.querySelector(".o-autocomplete-value-focus .o-autocomplete-value")!.textContent
       ).toBe("SUM");
     });
+
+    test("key down or up selects auto-complete proposals instead of reference", async () => {
```

Comment:
wasn't this already tested somewhere?

Reply:
I don't think it was possible before to be, at the same time, in a position where you can select references with the arrow keys AND with the autocomplete drop down open.